server/chat-plugins/repeats.ts

```diff
@@ -75,7 +78,9 @@ export const Repeats = new class {
 					this.clearRepeats(room);
 					return;
 				}
-				room.add(`|html|<div class="infobox">${repeat.phrase}</div>`);
+				const phrase = !repeat.faq ? repeat.phrase :
+					Chat.formatText(roomFaqs[room.roomid][repeat.faq], true).replace(/\n/g, '<br />');
```

Comment:
```suggestion
				const phrase = repeat.faq ? Chat.formatText(roomFaqs[room.roomid][repeat.faq], true).replace(/\n/g, '<br />') :
					repeat.phrase;
```
Personally, I think it would be more readable to make `Chat.formatText(..., true).replace(/\n/g, '<br />');` a function, since it's kind of unclear that it's transforming the rfaq text into something that we can repeat, but that's up to you.

Reply:
i think that should be handled in another PR since it's used within numerous files so it's better as a utils/chat function